react/features/jwt/reducer.js

```diff
@@ -0,0 +1,22 @@
+import { ReducerRegistry } from '../base/redux';
+
+import { SET_TOKEN_DATA } from './actionTypes';
+
+/**
+ * Listen for actions that update the state of token data part of the Redux
+ * store.
+ *
+ * @param {Object} state - Current state.
+ * @param {Object} action - Action object.
+ * @returns {Object}
+ */
+ReducerRegistry.register('features/jwt', (state = {}, action) => {
+    switch (action.type) {
+    case SET_TOKEN_DATA:
+        return {
+            ...action.tokenData
```

Comment:
Storing the object itself will make handling this a lot simpler. I don't want to rely on an instance of TokenData where all the fields are undefined + destructuring.